src/Serializing/SExpression/Precise.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE DataKinds, FlexibleContexts, FlexibleInstances, MultiParamTypeClasses, OverloadedStrings, ScopedTypeVariables, TypeApplications, TypeFamilies, TypeOperators, UndecidableInstances #-}
+{-# LANGUAGE AllowAmbiguousTypes, DataKinds, FlexibleContexts, FlexibleInstances, MultiParamTypeClasses, OverloadedStrings, ScopedTypeVariables, TypeApplications, TypeFamilies, TypeOperators, UndecidableInstances #-}
```

Comment:
Down with proxies!

Reply:
But at what `Ambiguous` cost??

Reply:
As with any case where I’m actually willing to use `AllowAmbiguousTypes`, it doesn’t escape the module 😊